pkg/sql/sqlbase/structured.go

```diff
@@ -835,6 +836,85 @@ func (desc *TableDescriptor) AllActiveAndInactiveChecks() []*TableDescriptor_Che
 	return checks
 }
 
+// ReplaceColumnVarsInExprWithDummies replaces the occurrences of column names in an expression with
+// dummies containing their type, so that they may be typechecked. It returns
+// this new expression tree alongside a set containing the ColumnID of each
+// column seen in the expression.
+func (desc *TableDescriptor) ReplaceColumnVarsInExprWithDummies(
```

Comment:
Why did you move this to `sqlbase` when it's only used in `schemaexpr`?

Reply:
hmm, this seems like an artifact of rebasing my changes onto the new schema expr package. I'd be happy to move it over, or you can do that if you are making some related changes.

Reply:
i'll go ahead and move it back in #49819